documentation/airport-guides/kstl.md

```diff
@@ -0,0 +1,67 @@
+
+### Airspace
+The T75 airspace encompasses a large area extending 40nm from St. Louis Lambert Int'l (STL), from the surface up to and including 15,000. The T75 TRACON also controls the Scott Shelf, located to the Southeast, up to 6,000ft. Class B airspace exists with tops of 8,000 over the Greater St. Louis area.
+
+### Instrument Procedures
+Most of the procedures are RNAV and contain both speed and altitude restrictions, taking a lot of weight off the controllers. Arriving aircraft must still be vectored to their final approach course.
+
+### Sectors Available in OpenScope
+_To be completed. Currently still in search of information on T75 sectorization._
+```
+F - Final  (default)
+N - North Arrival  (unavailable)
+S - South Arrival  (unavailable)
+D - Departure      (unavailable)
+L - Low/Satellite  (unavailable)
+```
+
+### Airports of Interest
+
+#### KSTL (Primary Airport)
+St. Louis Lambert International  
+194,173 annual operations
+
+Originally a balloon launching base, Lambert Field became the first airport with an air traffic control system; The first controller, Archie League, communicated with pilots by waving flags. Since then, the airport has grown and become the busiest airport in Missouri, with three main parallel runways and one crosswind runway. In calm winds, runway 30R/12L is the main arrival runway and 30L/12R the primary departure runway. In heavy traffic, runway 29/11 is also used for both departures and arrivals. Simulatenous approaches are permitted to runways 30R/12L and 29/11.
+
+#### KSUS
+Spirit of St. Louis Airport  
+118,452 annual operations  
+14.2nm Southwest of KSTL
+
+The airport is named after the single engine aircraft "Spirit of St. Louis", flown by Charles Lindbergh in 1927, on the first solo non-stop transatlantic flight from New York to Paris. The airport has two parallel runways and is used primarily for general and corporate aviation.
+
+#### KCPS
+St. Louis Downtown Airport  
+109,557 annual operations  
+14.7nm Southeast of KSTL
+
+Located only 1 mile from downtown St. Louis, the airport is used by many business aircraft visiting the region.
+
+#### KSET
+St. Charles County Smartt Airport  
+60,720 annual operations  
+11.2nm North of KSTL
+
+Used as a Navy Training site during World War II, Smartt field is now a small airport used almost exclusively for general aviation.
+
+#### KALN
+St. Louis Regional Airport  
+39,828 annual operations  
+17.4nm Northeast of KSTL
+
+St. Louis Regional is a small general aviation airport located 4 miles east of Alton, Illinois.
+
+#### KBLV
+MidAmerica St. Louis Airport / Scott AFB  
+25,854 annual operations  
+27.9nm Southeast of KSTL
+
+MidAmerica Airport is co-located on the grounds of Scott Air Force Base; The Northern side is used for civilian operations and Southern side is owned by the U.S. Air Force. It was originally built in 1997 as a reliever airport for STL.
+
+### Additional Resources
+[VATSIM SOP](http://kcartcc.com/open/38)
```

Comment:
Is this link still working?

Reply:
The whole site is down and I don't think they moved it so I'm guessing something went terribly wrong and they're working on it. I'd say we leave it there for now in case it comes back to life.